site/src/sphinx/client-retry.rst

```diff
@@ -18,33 +18,18 @@ So, let's find out what we can do with :api:`RetryingClient`.
 ``RetryingClient``
 ------------------
 
-You can just use the ``decorator()`` method in :api:`ClientBuilder` to build a :api:`RetryingHttpClient`:
+You can just use the ``decorator()`` method in :api:`HttpClientBuilder` to build a :api:`RetryingHttpClient`:
```

Comment:
How about:

```rst
... in :api:`ClientBuilder` or :api:`HttpClientBuilder` ...
```

.. just in case it's unclear?

Reply:
Hmm,, let me modify your intention on commit.